Rakefile

```diff
@@ -11,7 +11,8 @@ Vmdb::Application.load_tasks
 begin
   require 'jasmine'
   load 'jasmine/tasks/jasmine.rake'
-rescue LoadError
+rescue LoadError => e
+  puts e.message
```

Comment:
@himdel @martinpovolny Do you have suggestions here?  I assume we did this so you can load the Rakefile in environments without jasmine.